lib/core/src/recover/recoverer.rs

```diff
@@ -142,7 +142,7 @@ impl Recoverer {
             .ok_or_else(|| anyhow!("Found no input for claim tx"))?;
 
         let script_witness_bytes = input.clone().witness.script_witness;
-        log::info!("Found Send Swap {swap_id} claim tx witness: {script_witness_bytes:?}");
+        info!("Found Send Swap {swap_id} claim tx witness: {script_witness_bytes:?}");
```

Comment:
It seems that If we can compare here the cached index with the wallet index then we can run a scan here and we won't need the full scan in the recovery